rm-datasource/src/main/java/io/seata/rm/datasource/exec/AbstractDMLBaseExecutor.java

```diff
@@ -172,4 +173,13 @@ public static boolean isLockRetryPolicyBranchRollbackOnConflict() {
             return LOCK_RETRY_POLICY_BRANCH_ROLLBACK_ON_CONFLICT;
         }
     }
+
+    protected void isUpdatePkValue(List<String> updateColumns) {
```

Comment:
return boolean? control of exception by caller?

Reply:
I think this is unnecessary. It just checks if the param is legal. There is nothing to control by the caller.

Reply:
this method is more suitable placed in UpdateExecutor or MultiUpdateExecutor. or rename method name.

Reply:
What method name do you want? Please give me some advice

Reply:
how about `assertContainsPKColumnName`?